ui/src/index.tsx

```diff
@@ -2,16 +2,11 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
 
 ReactDOM.render(
-  <React.StrictMode>
```

Comment:
Why were the `StrictMode` tags removed?

Reply:
StrictMode would cause the component to run twice and it is just for development. There is no effect to leave that out